static/app/views/detectors/new.tsx

```diff
@@ -0,0 +1,34 @@
+import {Button, LinkButton} from 'sentry/components/button';
+import {Flex} from 'sentry/components/container/flex';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {BreadcrumbsProvider} from 'sentry/components/workflowEngine/layout/breadcrumbs';
+import EditLayout from 'sentry/components/workflowEngine/layout/edit';
+import {
+  StickyFooter,
+  StickyFooterLabel,
+} from 'sentry/components/workflowEngine/ui/footer';
+import {useWorkflowEngineFeatureGate} from 'sentry/components/workflowEngine/useWorkflowEngineFeatureGate';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+export default function DetectorNew() {
```

Comment:
🤔 is it worth us making a component for `new` and `new settings` then pass in the label `montiors` or will we be extending these in super different directions? (my though is probably not, but wanted to verify)

Reply:
That's actually a great call, I can make this a little more composable